### DISCUSSION TITLE
posts/membuat-vite-agar-mendukung-browser-lama

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#101] [General]
# posts/membuat-vite-agar-mendukung-browser-lama

Menambahkan plugin legacy pada vite agar hasil build mendukung browser lama

https://teknologiumum.com/posts/membuat-vite-agar-mendukung-browser-lama

**Comment by mzaini30** [comment]
Untuk Chrome versi 65 kayaknya nggak ngefek ya?
